code/datums/station_traits/neutral_traits.dm

```diff
@@ -71,10 +72,25 @@
 	weight = 5
 	show_in_report = TRUE
 	report_message = "We lost the primary datatape that holds the announcement system's voice responses. We did however find an older backup."
-	blacklist = list(/datum/station_trait/announcement_intern,
+	blacklist = list(/datum/station_trait/announcement_intern, /datum/station_trait/announcement_ancestor,
 	/datum/station_trait/announcement_medbot
 	)
 
 /datum/station_trait/announcement_baystation/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/baystation
+
+
+/datum/station_trait/announcement_ancestor
+	name = "Announcer: Ancestor"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 1
```

Comment:
This makes it as rare if not rarer than joke intern announcer. I'd suggest bumping it to 2 or 3.

Reply:
i wanted the trait to be incredibly rare, so people wouldnt get bored of it so soon.